app/src/dhis/java/org/dhis2/Bindings/ContextExtensions.kt

```diff
@@ -5,15 +5,14 @@ import android.content.Context
 import android.content.pm.PackageManager
 import androidx.core.content.ContextCompat
 import androidx.fragment.app.Fragment
-import hu.supercluster.paperwork.Paperwork
 import org.dhis2.BuildConfig
 
 fun Context.buildInfo(): String {
     val paperWork = Paperwork(this)
```

Comment:
Where does this `Paperwork` class come from?

Reply:
from the library that was removed